notify/impl.go

```diff
@@ -584,7 +584,7 @@ func (n *Hipchat) Notify(ctx context.Context, as ...*types.Alert) error {
 	}
 
 	req := &hipchatReq{
-		From:          tmplText(n.conf.From),
+		From:          truncate(tmplText(n.conf.From), 64),
```

Comment:
This makes me realize, we should also validate the other fields. We are going down quite the rabbit hole once we start getting into stuff like valid `color` fields etc.

I also cannot infer whether "length" refers to the number of (unicode) runes or the bytes in the encoded JSON string.